modules/mysql/variables.tf

```diff
@@ -161,6 +167,14 @@ variable "user_labels" {
   description = "The key/value labels for the master instances."
 }
 
+variable "data_cache_config" {
```

Comment:
Since there is only one value can you just change it to string variable:

```
variable "data_cache_enabled" {
  description = "Whether data cache is enabled for the instance. Defaults to false. Feature is only available for ENTERPRISE_PLUS tier and supported database_versions"
  type        = bool
  default     = false
}
```

Reply:
Done